spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -777,9 +777,12 @@ def main(argv: Sequence[str]):
         printv('\nRemove unused label on template. Keep only label present in the input label image...', verbose)
         sct_labels.remove_missing_labels(Image(ftmp_template_label), Image(ftmp_label)).save(path=ftmp_template_label)
 
-        # Add one label because at least 3 orthogonal labels are required to estimate an affine transformation.
-        add_orthogonal_label(ftmp_label)
-        add_orthogonal_label(ftmp_template_label)
+        # Add a dummy label, because at least 3 orthogonal labels are required to estimate an affine transformation.
+        # -> Pick a dummy label >=99 that doesn't clash with the existing label values.
+        existing_label_vals = {coord.value for coord in img_tmp_label.getCoordinatesAveragedByValue()}
+        dummy_label = 99 if 99 not in existing_label_vals else max(existing_label_vals) + 1
```

Comment:
Another reason for moving the dummy label value determination to the outer `sct_register_to_template` context is that it ensures we use the same value for both images. _(It felt a little risky to programmatically determine the label twice in two separate calls to `add_orthogonal_label`.)_

(I think it should be safe to use the labels from `img_tmp_label`? Since, any labels in the template label image that aren't present in `img_tmp_label` get removed earlier on.)